.circleci/config.yml

```diff
@@ -67,7 +67,7 @@ jobs:
               fi
       - run:
           name: Deploy
-          command: ./deploy.sh
+          command: ./gradlew :app:uploadDeployGateDebug
```

Comment:
Maybe `./gradlew uploadDeployGateRelease` is correct.
Plus, this command should be run when the code is merged to only master branch.
https://github.com/DroidKaigi/conference-app-2018/blob/master/deploy.sh#L3

Reply:
Can you add the branch check logic like this?
https://github.com/DroidKaigi/conference-app-2018/pull/204/files#diff-1d37e48f9ceff6d8030570cd36286a61R63